### DISCUSSION TITLE
Add CLIP integrity check to the model verification scans.

### DISCUSSION THREAD

**Discussion by Machinekng** [civitai/civitai#208] [Ideas]
With the release of model some new model tools, folks in the community have noticed that their own merges and some popular models have issues with the CLIP component. Having a broken CLIP can lead to the model ignoring parts of the prompt, making it less reliable. 

Several CLIP checking extensions have been implemented in popular UIs, but I think it would be helpful to add this sort of check to civitai as well in case model creators aren't aware of this issue and these tools. I'm not sure how easy it would be to implement, but I assume it can be implemented in a similar way to the pickle checker.

Rentry page on CLIP fix: https://rentry.org/clipfix

**Comment by JustMaier** [comment]
I think it'd be best if we could just automatically fix a broken CLIP. Have you or has anyone you know gone through the CLIP fix process before? It'd be good to talk with them a bit about the process to see if we can find a way to automate the fix process.
